nemo/collections/nlp/modules/common/megatron/transformer.py

```diff
@@ -1428,7 +1431,7 @@ def forward(
             # fp8_autocast will not do anything if TE or FP8 isn't used
             fp8_group = None
             if self.fp8 and parallel_state.model_parallel_is_initialized():
-                fp8_group = parallel_state.get_amax_reduction_group()
+                fp8_group = parallel_state.get_data_parallel_group()
```

Comment:
I don't think this will be quite right in the case we have tensor-parallelism. How about the following to make clear this is a hack:
```suggestion
                # TODO @tmoon Restore once available in Megatron-LM
                # fp8_group = parallel_state.get_amax_reduction_group()
                fp8_group = parallel_state.get_amax_reduction_group() if hasattr(parallel_state, 'get_amax_reduction_group') else parallel_state.get_data_parallel_group()
```

Reply:
Sure, I've adopted the suggestion.